components/AppHeader.vue

```diff
@@ -48,39 +35,9 @@ export default {
     }
   },
   methods: {
-    openTechnicalPaper() {
-      window.open(
-        'https://www.notion.so/Arcana-Technical-Docs-a1d7fd0d2970452586c693e4fee14d08',
-        '_blank'
-      )
+    handleOpenDashboard() {
+      window.open('https://dashboard.arcana.network', '_blank')
     },
```

Comment:
We should avoid using `window.open` for opening external links and create proper `a` tags instead. Possibly something for you to refactor in the `VHeader` component.